build.py

```diff
@@ -180,13 +168,15 @@ def main():
         nargs='+',
         default=[
             'org.gel.models.participant.avro::1.0.0', 'org.gel.models.participant.avro::1.0.1',
-            'org.gel.models.participant.avro::1.0.2',
+            'org.gel.models.participant.avro::1.0.2', 'org.gel.models.participant.avro::1.0.3-SNAPSHOT',
```

Comment:
`org.gel.models.participant.avro::1.0.3-SNAPSHOT` should be changed to `org.gel.models.participant.avro::1.0.3` and `org.gel.models.participant.avro::1.0.4-SNAPSHOT` needs to be added